converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java

```diff
@@ -308,6 +310,102 @@ private void createTransformation(final Element rules, final Mapping mapping) th
 		processTransformationComponentFunction(transformationComponent, mapping, mappingInputsVariablesMap, rules);
 	}
 
+	private void createLookupTable(final Element maps, final Component transformationComponent) throws DMPConverterException {
+
+		if (transformationComponent == null)
+			return;
+
+		final Function transformationFunction = transformationComponent.getFunction();
+
+		if (transformationFunction != null && transformationFunction.getFunctionType() == FunctionType.Transformation) {
+
+			final Transformation transformation = (Transformation) transformationFunction;
+
+			final Set<Component> components = transformation.getComponents();
+
+			if (components == null)
+				return;
+
+			for (final Component component : components) {
+
+				final Map<String, String> componentParameterMapping = component.getParameterMappings();
+
+				if (LOOKUP_FUNCTIONS.contains(component.getFunction().getName()) && componentParameterMapping != null) {
+
+					final Element map = doc.createElement("map");
```

Comment:
we may should "outsource" those identifiers ("map", "maps", "rules", ...) to static final attributes of the MSB, or?